met-api/src/met_api/models/survey.py

```diff
@@ -56,6 +56,13 @@ def get_all_surveys(cls) -> List[SurveySchema]:
         data = db.session.query(Survey).join(Engagement, isouter=True).join(EngagementStatus, isouter=True).all()
         return survey_schema.dump(data)
 
+    @classmethod
+    def get_all_unlinked_surveys(cls) -> List[SurveySchema]:
+        """Get all surveys that are unlinked to engagement."""
+        survey_schema = SurveySchema(many=True)
+        data = db.session.query(Survey).filter_by(engagement_id=None).all()
```

Comment:
just a suggestion , call it surveys instead of data ...easily comprehensible that way

Reply:
noted, will do